core/src/test/java/com/graphhopper/reader/dem/GraphElevationSmoothing_DiffblueTest.java

```diff
@@ -0,0 +1,72 @@
+/**
+ *
+ *
+ * Copyright 2018 Diffblue Limited
+ *
+ * Diffblue Limited licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.graphhopper.reader.dem;
+
+import static org.mockito.Matchers.anyInt;
+
+import com.graphhopper.reader.dem.GraphElevationSmoothing;
+import com.graphhopper.util.DistancePlaneProjection;
+import com.graphhopper.util.PointList;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.api.mockito.expectation.PowerMockitoStubber;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.lang.reflect.Method;
+
+@RunWith(PowerMockRunner.class)
+public class GraphElevationSmoothing_DiffblueTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: GraphElevationSmoothing */
+
+  /*
+   * Test generated by Diffblue Deeptest.This test covers
+   * `com.graphhopper.util.PointList
+   * smoothElevation(com.graphhopper.util.PointList)' block 36 (line 47)
+   *
+   */
+  @PrepareForTest({DistancePlaneProjection.class, PointList.class})
+  @Test
```

Comment:
👍

Reply:
👎  PointList shouldn't be mocked. There's no reasonable check on the return value.